scripts/create-new-actor.sh

```diff
@@ -137,35 +139,31 @@ create_impl_text() {
     echo "struct ${actor_name}::PhaseConfig {"
     echo "    mongocxx::collection collection;"
     echo "    DocumentGenerator documentExpr;"
-    echo "    ExecutionStrategy::RunOptions options;"
     echo ""
     echo "    PhaseConfig(PhaseContext& phaseContext, const mongocxx::database& db, ActorId id)"
     echo "        : collection{db[phaseContext.get<std::string>(\"Collection\")]},"
-    echo "          documentExpr{phaseContext.createDocumentGenerator(id, \"Document\")},"
-    echo "          options{ExecutionStrategy::getOptionsFrom(phaseContext, \"ExecutionsStrategy\")} {}"
+    echo "          documentExpr{phaseContext.createDocumentGenerator(id, \"Document\")} {}"
     echo "};"
     echo ""
     echo "void ${actor_name}::run() {"
     echo "    for (auto&& config : _loop) {"
     echo "        for (const auto&& _ : config) {"
-    echo "            _strategy.run("
-    echo "                [&](metrics::OperationContext& ctx) {"
-    echo "                    // TODO: main logic"
-    echo "                    auto document = config->documentExpr();"
-    echo "                    BOOST_LOG_TRIVIAL(info) << \" ${actor_name} Inserting \""
-    echo "                                            << bsoncxx::to_json(document.view());"
-    echo "                    config->collection.insert_one(document.view());"
-    echo "                    ctx.addDocuments(1);"
-    echo "                    ctx.addBytes(document.view().length());"
-    echo "                },"
-    echo "                config->options);"
+    echo "            auto inserts = _inserts.start();"
```

Comment:
Do you think `s/_inserts/_totalInserts` would be more intuitive? I fear the lack of anything mentioning "metrics" in this file could lead people down a rat hole of poking around in `context.operation()`.